packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -66,6 +97,20 @@ abstract class ProgressIndicator extends StatefulWidget {
     super.debugFillProperties(properties);
     properties.add(PercentProperty('value', value, showName: false, ifNull: '<indeterminate>'));
   }
+
+  Widget _buildSemanticsWrapper({
+    @required BuildContext context,
+    @required Widget child,
+  }) {
+    return Semantics(
+      container: true,
+      liveRegion: true,
```

Comment:
I would have expected that you can control if you want to be a live region (= get constant audio-updates about the state of the refresh indicator) or not (= state is only announced once when the user moves a11y focus to this node).